examples/connext_dds/network_capture/01_shared_memory_and_udp/c++11/network_capture_publisher.cxx

```diff
@@ -12,11 +12,12 @@
 
 #include <iostream>
 
-#include "application.hpp"  // for command line parsing and ctrl-c
-#include "network_capture.hpp"
 #include <dds/pub/ddspub.hpp>
-#include <rti/config/Logger.hpp>  // for logging
 #include <rti/util/util.hpp>      // for sleep()
+#include <rti/config/Logger.hpp>  // for logging
```

Comment:
**Important!**: I must have changed the ordering of these includes before merging the examples. Consequence: the current C++11 example is broken in develop. Merging this PR solves the problem.

Reply:
This will be fixed in separate [PR](https://github.com/rticommunity/rticonnextdds-examples/pull/359).